### DISCUSSION TITLE
Doc comments in line above?

### DISCUSSION THREAD

**Discussion by Kenionatus** [BeeStation/NSV13#2020] [General]
I would like to know the official stance on doc comments in the following format:
```
///comment for foo
var/foo = "bar"
///comment for bat
var/bat = "bazing"
```

Are they
1. recommended
2. recommended against
3. one of the above and not to be inserted into new code or
4. disallowed
5. depends, maintainers will decide case by case

(didn't even add "required" cause I know that's definitely not the case)

**Comment by Kenionatus** [comment]
Specifically asking
@covertcorvid
@Bokkiewokkie
and
@Karmic-Skink

**Comment by Bokkiewokkie** [comment]
I thought we already told you, but anyways, just don't do this. It makes the files more cluttered and is inconsistent with the rest of the code.
If you need to add a multiline comment anywhere for some reason it should be above the code it's describing, otherwise just keep it on the same line.

**Reply by Kenionatus** [reply]
Thanks for the clarification.
